examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        multiple_gate_set = ["cnot", "ccnot"]
</pre><p>Same goes here; put the gate instances in the list directly:</p><pre class="ql-syntax" spellcheck="false">multiple_gate_set = [gates.CNot(), gates.CCNot()]
</pre><p>and add them to the circuit:</p><pre class="ql-syntax" spellcheck="false">circuit.add_instruction(Instruction(gate, qubits))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>